### DISCUSSION TITLE
Build fails due to a type reference to a C++/CLI project

### DISCUSSION THREAD

**Discussion by gantonopoulos** [stryker-mutator/stryker-net#1933] [Q&A]
Hello,
stryker fails to build, reporting and error  CS0103 on types that come from a .dll produced by a C++/CLI project.
Does that mean the stryker does not support C++/CLI projects?

**Comment by rouke-broersma** [comment]
Stryker is a mutation testing tool for csharp, it does not work with c++.

**Reply by gantonopoulos** [reply]
Thank you for the answer.  So is the mutation already taking place during build? Because there is no reason the build should fail.
If yes, will excluding the files that contain these references from mutation work?

**Reply by rouke-broersma** [reply]
I do not know sorry, this scenario has not been tested by me.
